packages/terra-enzyme-intl/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Fixed
   * Fixed jest test failure caused by react-intl update.
 
+* Changed
+    * Updated jest tests.
```

Comment:
The changes in terra-enzyme can be removed since it is now fixed in the main branch.

Reply:
Removed my changelog update, but it looks like we had 2 changelog updates in enzyme? one outside the mini area so fixed that https://github.com/cerner/terra-toolkit/pull/737/commits/156b9b8a63e4938ca1d82fa99bf2c739ebc95b38#diff-6a20a00306b3df5fce16cd36681d7cc3d8447ea8382c5f116291111797267838L11